app/src/main/java/com/ncnf/views/fragments/map/MapFragment.java

```diff
@@ -85,7 +89,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         mapView.getMapAsync(googleMap -> {
             mMap = googleMap;
 
-            mapHandler = new MapHandler((AppCompatActivity) requireActivity(), mMap, getChildFragmentManager(), eventRepository);
+            mapHandler = new MapHandler((AppCompatActivity) requireActivity(), mMap, getChildFragmentManager(), eventRepository, organizationRepository);
```

Comment:
Can't the organizationRepository be injected in the mapHandler?

Reply:
I think it needs to be here because of the MapFragmentTest, but as I'm not really an expert on those I'm not sure... I followed what Jeremie did